modules/logging-rn-5.6.0.adoc

```diff
@@ -22,9 +22,14 @@ In logging version 5.6, Fluentd is deprecated and is planned to be removed in a
 
 * With this update, the *Developer* role can access the per-project workload logs they are assigned to within the Log Console Plugin on clusters running {product-title} 4.11 and higher. (link:https://issues.redhat.com/browse/LOG-3388[LOG-3388])
 
-* With this update, logs from any source contain a field `openshift.cluster_id`, the unique identifier of the cluster in which the Operator is deployed. You can view the `clusterID` value with the command below. (link:https://issues.redhat.com/browse/LOG-2715[LOG-2715])
-
-include::snippets/logging-get-clusterid-snip.adoc[lines=9..12]
+* With this update, logs from any source contain a field `openshift.cluster_id`, the unique identifier of the cluster in which the Operator is deployed. You can view the `clusterID` value by using the following command:
```

Comment:
```suggestion
* With this update, logs from any source contain an `openshift.cluster_id` field, which is the unique identifier of the cluster in which the Operator is deployed. You can view the `clusterID` value by using the following command:
```

Reply:
I won't change the wording of this for now just because it's in release notes and that would maybe require change management, which I'd rather not have to wait for before we can merge this one due to the broken CP docs 🙂